integration-tests/sql/pom.xml

```diff
@@ -143,6 +139,125 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>h2</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>!SQL_JDBC_DB_KIND</name>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-jdbc-h2</artifactId>
+                </dependency>
+            </dependencies>
+        </profile>
+        <profile>
+            <id>postgresql</id>
+            <activation>
+                <property>
+                    <name>SQL_JDBC_DB_KIND</name>
+                    <value>postgresql</value>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-jdbc-postgresql</artifactId>
```

Comment:
I wonder if some thing like quarkus-jdbc-${SQL_JDBC_DB_KIND} could be used. It would be less lines in the pom file and maybe even fail when a wrong value is provided. I'm just curious, it's not blocking at all.

Reply:
I started with this approach (similar to  quarkus-jdbc-${SQL_JDBC_DB_KIND} ) , I'll try to return to it

Reply:
done